src/ansys/mapdl/core/_commands/post26/special_purpose.py

```diff
@@ -0,0 +1,588 @@
+            the command is manually typed in.
+
+        pcnam2 : str
+            Names of element components for power loss calculation. Must be enclosed in single quotes when
+            the command is manually typed in.
+
+        ecnam1 : str
+            Names of element components for energy and total current calculations, respectively. Must be
+            enclosed in single quotes when the command is manually typed in.
+
+        ccnam1 : str
+            Names of element components for energy and total current calculations, respectively. Must be
+            enclosed in single quotes when the command is manually typed in.
+
+        Notes
+        -----
+
+        .. _PMGTRAN_notes:
+
+        :ref:`pmgtran` invokes a Mechanical APDL macro which calculates and summarizes electromagnetic
+        results from
+        a transient analysis. The results are summarized by element components and listed on the screen as
+        well as written to a file ( :file:`Fname.OUT` ).
+
+        You can select two components for the summary of electromagnetic forces, two for power loss, and one
+        each for stored energy (see :ref:`senergy` ) and total current (see :ref:`curr2d` ). See the
+        referenced commands for other restrictions.
+
+        :ref:`pmgtran` is restricted to MKSA units.
+        """
+        command = f"PMGTRAN,{fname},{freq},{fcnam1},{fcnam2},{pcnam1},{pcnam2},{ecnam1},{ccnam1}"
+        return self.run(command, **kwargs)
+
+    def vget(
+        self,
+        par: str = "",
+        ir: str = "",
+        tstrt: str = "",
+        kcplx: int | str = "",
+        **kwargs,
+    ):
+        r"""Moves a variable into an array parameter vector.
+
+        Mechanical APDL Command: `VGET <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_VGET.html>`_
+
+        Parameters
+        ----------
+        par : str
+            Array parameter vector in the operation.
+
+        ir : str
+            Reference number of the variable (1 to NV ( :ref:`numvar` )).
+
+        tstrt : str
+            Time (or frequency) corresponding to start of ``IR`` data. If between values, the nearer value
+            is used.
+
+        kcplx : int or str
+            Complex number key:
+
+            * ``0`` - Use the real part of the ``IR`` data.
+
+            * ``1`` - Use the imaginary part of the ``IR`` data.
+
+        Notes
+        -----
+
+        .. _VGET_notes:
+
+        Moves a variable into an array parameter vector. The starting array element number must be defined.
+        For example, :ref:`vget`,A(1),2 moves variable 2 (starting at time 0.0) to array parameter A.
+        Looping continues from array element A(1) with the index number incremented by one until the
+        variable is filled. The number of loops may be controlled with the `\*VLEN
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_VLEN.html#>`_  :ref:`vlen`
```

Comment:
Shall we separate both command versions with a `/`??
```suggestion
        variable is filled. The number of loops may be controlled with the `\*VLEN
        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_cmd/Hlp_C_VLEN.html#>`_  / :ref:`vlen`
```